src/game/server/neo/bot/neo_bot.cpp

```diff
@@ -1842,7 +1842,7 @@ bool CNEOBot::IsLineOfFireClear(const Vector& from, CBaseEntity* who, const Line
 	const auto lofMask = LineOfFireMask(flags);
 	UTIL_TraceLine(from, to, lofMask, &filter, &trace);
 
-	const bool bIsClear = !trace.DidHit() || trace.m_pEnt == who;
+	const bool bIsClear = !trace.DidHit() || trace.m_pEnt == who || (trace.DidHit() && IsAbleToBreak(trace.m_pEnt));
```

Comment:
Ditto on the `IF NOT TRUE || TRUE && ...` pattern regarding the `DitHit...` case for simplifying the syntax where the `||` already implies the opposite